### PR TITLE
RELATED: FET-807 Upgrade packages

### PR DESCRIPTION
**Pull request by pbenes**
* Upgrade packages
* Fix screenshots (latest mock-js)

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8365/ : SUCCESS in 5m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4734/ : SUCCESS in 7s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4412/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4735/ : SUCCESS in 48s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3790/ : SUCCESS in 12m 38s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by pbenes** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1833.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8368/ : SUCCESS in 5m 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4738/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4739/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3791/ : SUCCESS in 18m 41s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4413/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4740/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3792/ : SUCCESS in 11m 30s

File: yarn.lock [hunk 1]
```diff
@@ -1239,13 +1239,20 @@
...
 
-"@gooddata/typings@^2.23.0", "@gooddata/typings@^2.26.1", "@gooddata/typings@^2.27.0":
+"@gooddata/typings@^2.26.1", "@gooddata/typings@^2.27.0":
   version "2.27.0"
   resolved "https://registry.yarnpkg.com/@gooddata/typings/-/typings-2.27.0.tgz#361888304d56d124cabd9625e81a3b0dae0597f8"
   integrity sha512-vB9/BQHtWHZtTl/LgO/Hsl8xr8YlmHy61kbj7A316Ana08H2wnqV8Y8ZjlVf94bHq9Hza7ViF7urhbCbfP8PFA==
   dependencies:
     lodash "^4.17.11"
 
+"@gooddata/typings@^2.28.0":
```
Comment (no23reason) [inline yarn.lock:1249]: Please deduplicate the typings pacakge

Comment (pbenes) [reply yarn.lock]: Fix pushed (it deduplicated also babel/runtime package)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8369/ : SUCCESS in 5m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4741/ : SUCCESS in 6s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-react-components-gate-PR1833

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/288/ : FAILURE in 2m 27s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/2073/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4414/ : SUCCESS in 11m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-sonarqube-scan-zuul-gate/55/ : SUCCESS in 4m 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8370/ : SUCCESS in 2m 52s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/3169/ : FAILURE in 20m 21s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-react-components-gate-PR1833

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#8370](https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8370/))
Covered lines: **91.99%**, change: **0%**
Covered branches: **84.04%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8371/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/289/ : FAILURE in 2m 47s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/2074/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4415/ : SUCCESS in 10m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-sonarqube-scan-zuul-gate/56/ : SUCCESS in 5m 34s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/8371/ : SUCCESS in 6m 06s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/3170/ : SUCCESS in 8m 03s

**Pull request merged**